File: model/genre_tracks.go
package model

import (
	"github.com/jinzhu/gorm"
)

type GenreTracks struct {
	gorm.Model
	GenreID int64 `gorm: "column: genre_id; type: bigint; primary_key; auto_increment: false; unique; not null"`
	TrackID int64 `gorm: "column: track_id; type: bigint; primary_key; auto_increment: false; unique; not null"`
}

File: admin/backend/src/db/model/genre-tracks.ts
import { Model, snakeCaseMappers, } from 'objection';

export class GenreTracks extends Model {
    genreId: string;
    trackId: string;

    static get tableName() {
        return 'genre_tracks';
    }

    static get idColumn() {
        return ['genre_id', 'track_id'];
    }

    static get columnNameMappers() {
        return snakeCaseMappers();
    }
}

export default GenreTracks;
